agenda/domain/src/main/java/com/nibalk/tasky/agenda/domain/model/AgendaDeletedItems.kt

```diff
@@ -0,0 +1,7 @@
+package com.nibalk.tasky.agenda.domain.model
+
+data class AgendaDeletedItems (
+    val eventIds: List<String>,
+    val taskIds: List<String>,
+    val reminderIds: List<String>
+)
```

Comment:
Do you need this object in domain? In the end you probably keep a reference to locally deleted agenda items in your DB right? Then you could just read them in the data layer and wouldn't need to pass any info to it.